getting_started/step_by_step/your_first_game.rst

```diff
@@ -746,6 +746,9 @@ function to set everything up for a new game:
 
 .. tabs::
  .. code-tab:: gdscript GDScript
+ 
+    func _on_Player_hit():
+         game_over()
 
```

Comment:
I don't think this is a good addition. It's extra code for no reason.

It's good for new learners to understand that when they connect a signal, they don't have to create a new function but can connect to an existing one.

Reply:
This is exactly the reason why I want to add the line. I want to show new learners what the code should look like after the connecting the signal. I missed this part when I ran through the tutorial and there was an error when running the game. Perhaps we can put these two lines separately to highlight this part. 

In general, I think it would be nice to have a way to show the entire script after each step (I am not sure how), because new learners like me are not sure if we are doing the right thing.